tinnsleep/scoring.py

```diff
@@ -122,40 +139,101 @@ def get_event_label(episode):
         return 3
 
 
-def create_list_events(li_ep, interval):
+def create_list_events(li_ep, time_interval, recording_duration):
     """ Creates the list of events, 0 = no event, 1 = tonic episode, 2 = phasic 
     episode, 3 = mixed episode
     
     Parameters
     ----------
     li_ep : list of episode instances
-    interval: float, interval between 2 elementary events
+    time_interval: float, time interval in seconds between 2 elementary events
+    recording_duration : float, duration of the recording in seconds
     Returns
     -------
     list of integers, labels of the events 
     """
-    #Deals the case the input is empty
+    # Deals the case the input is empty
     li_events = []
     if len(li_ep) == 0:
         return []
 
     # Initialization
     # Case where the first burst does not begin at time=0
-    if li_ep[0].beg / interval > 1:
-        li_0 = [0 for i in range(int(li_ep[0].beg / interval))]
+    if li_ep[0].beg / time_interval > 1:
+        li_0 = [0 for i in range(int(li_ep[0].beg / time_interval))]
```

Comment:
carefull using ``int()`` to round. It is actually always rounding to the inferior integer. I believe that in this case, it is the expected behaviour (you don't want to overflow the total length of the recording) but it is important to keep that in mind.

Using uglier but safer rounding it is use specific rounding function such as `ceil()` or `floor()` before converting to int.
Also `round()` or `around()` are very bad because it is a ugly statistician rounding to even (which is not the ISO rounding), read the Note of around()
https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.around.html#numpy.around

Reply:
I approve, I had bad experiences with round()